pkg/formatted/k8s.go

```diff
@@ -22,3 +22,10 @@ func Condition(c apis.Condition) string {
 
 	return status
 }
+
+func IsFailed(c apis.Condition) bool {
```

Comment:
imho, we shouldn't have functions that are not related to fomatting an input to return a string in this package.

Reply:
Also. `HasFailed` may improve readability